### PR TITLE
feat: 明细表的 getCellData 可获取单行数据

### PR DESCRIPTION
**Pull request by stone-lyl**
### 👀 PR includes

✨ Feature

- [ ] New feature

🎨 Enhance

- [ ] Code style optimization
- [ ] Refactoring
- [ ] Change the UI
- [ ] Improve the performance
- [ ] Type optimization

🐛 Bugfix

- [ ] Solve the issue and close #0

🔧 Chore

- [ ] Test case
- [ ] Docs / demos update
- [ ] CI / workflow
- [ ] Release version
- [ ] Other ()

### 🖼️ Screenshot
feat: 明细表的 getCellData 可获取单行数据
fix: 明细模式下返回值 对齐 聚合模式，添加 rowIndex

| Before | After |
| ------ | ----- |
| ❌      | ✅     |

### 🔍 Self-Check before the merge

- [ ] Add or update relevant docs.
- [ ] Add or update relevant demos.
- [ ] Add or update test case.
- [ ] Add or update relevant TypeScript definitions.

### CODE REVIEW

File: packages/s2-core/src/data-set/table-data-set.ts [hunk 1]
```diff
@@ -149,7 +149,13 @@ export class TableDataSet extends BaseDataSet {
     if (this.displayData.length === 0 && query.rowIndex === 0) {
       return;
     }
-    return this.displayData[query.rowIndex][query.col];
+
```
Comment (stone-lyl) [inline packages/s2-core/src/data-set/table-data-set.ts:152]: 交叉表业务需求，需要获取明细表一整行的数据。

**Review by lcx-seima — APPROVED** [review]
LG

**Review by xingwanying — COMMENTED** [review]
顺便给趋势图点击的事件也加上？
DATA_CELL_TREND_ICON_CLICK

**Comment by lijinke666** [comment]
感觉这里有个小细节可以优化下, 点击链接时, 不应该显示 Tooltip

image

**Pull request merged**